statsmodels/stats/diagnostic.py

```diff
@@ -835,8 +835,10 @@ def het_white(resid, exog):
         The residuals. The squared residuals are used as the endogenous
         variable.
     exog : array_like
-        The explanatory variables for the variance. Squares and interaction
-        terms are automatically included in the auxiliary regression.
+        The explanatory variables for the variance. Squares and, by default,
+        interaction terms are automatically included in the auxiliary regression.
+    interaction_terms : bool, default True
```

Comment:
Should probably be False for now since this would change the output of tests without a deprecation.